config.yaml

```diff
@@ -1,8 +1,8 @@
 default:
  log_file: "/var/log/host-os/builds.log"
  packages_repos_target_path: "/var/lib/host-os/repositories"
- build_versions_repository_url: "https://github.com/open-power-host-os/versions.git"
- build_version: 'master'
+ packages_metadata_repo_url: "https://github.com/open-power-host-os/versions.git"
```

Comment:
But that's exactly what this git repository holds, package metadata! "versions" is a confusing name because it seems like this git repo keeps only versions number and it keeps much more than that. "packages" is also a little confusing because people may confuse with the packages source code git repositories. I prefer a longer clearer name.

btw nothing prevents us from renaming the 'versions' git repository to 'packages' or 'packages_metadata'